css/src/components/segmented-control.scss

```diff
@@ -0,0 +1,131 @@
+$sc-padding-sm: 0.5em 1em !default;
+$sc-padding: 0.75em 1.25em !default;
+$sc-padding-lg: 1em 1.5em !default;
+
+$sc-font-color: $primary !default;
+$sc-border: $border-width solid $border !default;
+$sc-hover-background-color: $primary-background !default;
+$sc-hover-border: $border-width solid $border !default;
+$sc-hover-border-color: $primary-hover !default;
+$sc-active-hover-background-color: $primary-hover !default;
+$sc-active-hover-border-color: $primary-hover !default;
+$sc-active-background-color: $primary !default;
+$sc-active-border-color: $primary !default;
+$sc-active-color: $text-invert !default;
+
+.segmented-controls {
+	display: grid;
+	justify-content: center;
+	width: 100%;
+	grid-gap: $spacer-5;
+	grid-template-columns: max-content auto max-content;
+	font-size: $font-size-7;
+	white-space: nowrap;
+
+	&.segmented-lg {
+		font-size: $font-size-5;
+	}
+}
+
+.segmented-control-previous,
+.segmented-control-next {
+	display: flex;
+	align-items: center;
+	padding: $sc-padding-sm;
+	border: $sc-border;
+	border-radius: $border-radius-rounded;
+	background-color: $sc-active-background-color;
+	color: $sc-active-color;
+	font-weight: $weight-semibold;
+	cursor: pointer;
+
+	&:hover {
+		border-color: $sc-active-hover-border-color;
+		background-color: $sc-active-hover-background-color;
+
+		@include forced-colors {
+			border-color: Highlight;
+			background-color: Highlight;
+			color: HighlightText;
+			forced-color-adjust: preserve-parent-color;
+		}
+	}
+}
+
+.segmented-control-list {
+	grid-column: 2;
+	display: flex;
+	align-items: center;
+	justify-content: flex-start;
+	border: $sc-border;
+	border-radius: $border-radius-rounded;
+	overflow-x: hidden;
+}
+
+.segmented-control {
+	display: flex;
+	position: relative;
+	align-items: center;
+	justify-content: center;
+	padding: $sc-padding-sm;
+	border: none;
+	border-radius: $border-radius-rounded;
+	background-color: transparent;
+	color: $sc-font-color;
+	font-weight: $weight-semibold;
+	cursor: pointer;
+	appearance: none;
+	vertical-align: top;
+
+	@include focus-visible() {
+		outline-offset: -$focus-width;
```

Comment:
In every theme, I'm having difficulties making out the focus outline color against the button background.

Usually when we do a negative outline offset, we also set `outline-color: currentColor`, which might help in this case?

Reply:
Alternatively, if the default focus outline can be made to break out of the tablist, then we wouldn't need the negative outline offset, and I think that'd be much more visible.

Reply:
Focus outlines are looking better now - resolving